Release/bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceMenuMod/Modules/props.lua

```diff
@@ -393,7 +394,7 @@ function Props:DrawProps(props)
       if Props.activeProps[prop.uid] and Props.activeProps[prop.uid].handle and Props.activeProps[prop.uid].handle ~= '' then
         local propPos = Props.activeProps[prop.uid].handle:GetWorldPosition()
         local distanceFromPlayer = Util:VectorDistance(playerPos, propPos)
-        if Props.activeProps[prop.uid].handle ~= '' and distanceFromPlayer < 3 then
+        if Props.activeProps[prop.uid].handle ~= '' and distanceFromPlayer < 6 then
```

Comment:
Is 6 significant, or just some items are at that range? Either way, extracting to a named constant would help explain the check

Reply:
No, that was a thing I'd changed for myself so I could move a tatami while standing on the opposite corner. I'll remove it tonight :D